pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.es-es.md

```diff
@@ -289,6 +292,31 @@ Su interfaz Zimbra dispone de 2 menús de configuración:
 >>
 >> Para configurar los filtros, consulte la sección "[Filtros](#filters)" de esta documentación.
 >>
+> **Agenda y recordatorios**
```

Comment:
is it Agenda or Calendario in Zimbra ? If possible, I'd recommend to have a common term everywhere